src/daemon/instance_settings_handler.cpp

```diff
@@ -153,6 +155,32 @@ void update_disk(const QString& key, const QString& val, mp::VirtualMachine& ins
     }
 }
 
+bool is_bridged(const mp::VMSpecs& spec)
+{
+    for (const auto& network : spec.extra_interfaces)
+    {
+        if (network.auto_mode)
+            return true;
+    }
+
+    return false;
+}
+
+void update_bridged(const QString& key, const QString& val, mp::VirtualMachine& instance, mp::VMSpecs& spec)
+{
+    auto bridged = mp::BoolSettingSpec{key, "false"}.interpret(val) == "true";
+
+    if (!bridged && is_bridged(spec))
+    {
```

Comment:
Is it possible to elaborate the role between bridged boolean (it looks like is checking user's input) and is_bridged(spec) (looks like checking if the spec is bridged)?

Reply:
Oh, sorry if it is unclear. The idea was that adding a boolean to the VM specs data structure would tell us quickly if the VM has a bridged interface. But the boolean can be inferred from the `extra_interfaces` vector. If some of the elements of this vector had the `auto_mode` set to true, it means the VM has a bridged interface. Thus, we removed the boolean. This approach lets us to have many bridged interfaces at the same time.

Reply:
Oh, sorry if it wasn't clear. The `bridged` boolean is the value given by the user. The `is_bridged()` function returns a boolean stating if the VM has a bridged interface.

Reply:
I see, thanks.